apps/playground-web/src/components/pay/transaction-button.tsx

```diff
@@ -21,8 +22,9 @@ const nftContract = getContract({
 });
 
 const usdcContract = getContract({
-  address: "0x833589fCD6eDb6E08f4c7C32D4f71b54bdA02913",
-  chain: base,
+  // biome-ignore lint/style/noNonNullAssertion: its there
+  address: getDefaultToken(sepolia, "USDC")!.address,
+  chain: sepolia,
```

Comment:
meant to change this? also if its there then the type should correctly infer that, right?

Reply:
Also yes. Function is a bit loose right now, not hard typed. Fine for now tho